src/lints/function_missing.ron

```diff
@@ -46,10 +46,9 @@ SemverQuery(
     },
     error_message: "A publicly-visible function cannot be imported by its prior path. A `pub use` may have been removed, or the function itself may have been renamed or removed entirely.",
     per_result_error_template: Some("function {{join \"::\" path}}, previously in file {{span_filename}}:{{span_begin_line}}"),
-    witness: (
-        hint_template: Some(
+    witness: Some((
```

Comment:
Could you try `serde(flatten)` in addition to `serde(default)` here? It might make the `Some` unnecessary.

Reply:
I tried to add it and it broke deserialization of the outer `SemverQuery` field :face_with_diagonal_mouth:

Reply:
It's possible to use ron's `implicit_some` option to be able to specify it without the `Some`, but this would allow (but not require) this behavior on all `Option<T>` in the `SemverQuery`.  [branch link](https://github.com/suaviloquence/cargo-semver-checks/compare/witness...implicit-some)

Reply:
Oh neat, TIL about that extension. I think that's fine, yeah. Maybe good for a separate PR we can merge immediately?